apps/vs-code-react/src/themeProvider.tsx

```diff
@@ -35,9 +33,5 @@ export const ThemeProvider: React.FC<{ children: ReactNode }> = ({ children }) =
     return () => observer.disconnect();
   }, [observer]);
 
-  return (
-    <FluentThemeProvider theme={theme === 'dark' ? AzureThemeDark : AzureThemeLight}>
-      <FluentProvider theme={theme === 'dark' ? webDarkTheme : webLightTheme}>{children}</FluentProvider>
-    </FluentThemeProvider>
-  );
+  return <FluentProvider theme={theme === 'dark' ? webDarkTheme : webLightTheme}>{children}</FluentProvider>;
```

Comment:
Even though we've removed the v8 components from vscode we still have some in designer, just want to make sure that it's okay to still remove the v8 provider here.